package.json

```diff
@@ -3,53 +3,51 @@
   "version": "0.0.0",
   "private": true,
   "license": "Apache-2.0",
-  "packageManager": "yarn@1.22.19",
-  "workspaces": {
-    "packages": [
-      "packages/*"
-    ],
-    "nohoist": [
-      "**/@types/**",
-      "**/typedoc",
-      "**/typedoc-plugin-markdown",
-      "**/typedoc-plugin-mdn-links"
-    ]
-  },
+  "packageManager": "pnpm@7.24.2",
   "dependencies": {
-    "lerna": "3.16.4"
+    "lerna": "5.6.2"
```

Comment:
Any reason why this version of lerna was chosen?

The latest is 6.4.0 https://www.npmjs.com/package/lerna